app/lobby/client/lobby.js

```diff
@@ -37,12 +37,18 @@ function proposeGame(event) {
     return $(this).val()
   }).get()
 
+  kingdom_id = $('.kingdom-id').val()
```

Comment:
I'd like to move the kingdom ID validation over to this side of things to keep it consistent with the other checks. This will significantly simplify the server side code.

Here, I think I'd like to do

```kingdom_id = _.trim($('.kingdom-id').val())```